sdks/java/core/src/test/java/org/apache/beam/sdk/schemas/JavaBeanSchemaTest.java

```diff
@@ -216,6 +219,39 @@ public void testNullableFromRow() throws NoSuchSchemaException {
     assertNull(bean.getStringBuilder());
   }
 
+  /**
+   * [BEAM-11530] Java distinguishes between parameter annotations and type annotations. Therefore
+   * annotations declared without {@link java.lang.annotation.ElementType#TYPE_USE} can't be
+   * accessed through {@link Executable#getAnnotatedParameterTypes()}. Some {@code @Nullable}
+   * annotations like {@link org.apache.avro.reflect.Nullable} do not declare {@link
+   * java.lang.annotation.ElementType#TYPE_USE} which makes them parameter annotations once placed
+   * in front of a parameter.
+   *
+   * @see <a
+   *     href="https://stackoverflow.com/a/37587590/5896429">https://stackoverflow.com/a/37587590/5896429</a>
+   * @see <a
+   *     href="https://github.com/apache/beam/pull/13622">https://github.com/apache/beam/pull/13622</a>
+   */
```

Comment:
Thank you this works :)

nit: we don't need the link to the PR, it can be found through `git blame` or the jira
```suggestion
   */
```